Clarify DD_TRACE_SAMPLE_RATE for PHP Tracer (#11549)
DataDog/documentation: content/en/tracing/setup_overview/setup/php.md

**Note:** Enabling this option may compromise security. This option, standalone, does not pose a security risk. However, an attacker being able to exploit a vulnerability in PHP or web server may be able to escalate privileges with relative ease, if the web server or PHP were started with full capabilities, as the background threads will retain their original capabilities. Datadog recommends restricting the capabilities of the web server with the `setcap` utility.

`DD_TRACE_SAMPLE_RATE`
: **Default**: `null`<br>

> Sets the trace sampling rate between `0.0` (0%) and `1.0` (100%, recommended). A value of`1.0`, Tracing without Limits™, sends all of your traffic. [Configure the retention][19] within the Datadog app. For versions before `0.36.0`, the sampling rate parameter is `DD_SAMPLING_RATE`.

Reviewer: ```suggestion
Sets the trace sampling rate between `0.0` (0%) and `1.0` (100%, recommended). A value of `1.0`, Tracing without Limits™, sends all of your traffic. [Configure the retention][19] within the Datadog app. For versions before `0.36.0`, the sampling rate parameter is `DD_SAMPLING_RATE`.
```

> Sets the trace sampling rate between `0.0` (0%) and `1.0` (100%, recommended). `1.0` or Tracing without Limits™, allows you to send all of your traffic and retention can be [configured within the Datadog app][19]. For versions < `0.36.0`, the sampling rate parameter is `DD_SAMPLING_RATE`.

Reviewer: ```suggestion
Sets the trace sampling rate between `0.0` (0%) and `1.0` (100%, recommended). A value of`1.0`, Tracing without Limits™, sends all of your traffic. [Configure the retention][19] within the Datadog app. For versions before `0.36.0`, the sampling rate parameter is `DD_SAMPLING_RATE`.
```
